inrupt/pod-browser
# Feat/hamburger menu

# Hamburger menu for small screens
In this PR:
- Slight refactor of the `header` component: `mainNav` is now its own component
- Added a hamburger menu only visible in small screens

### To test:
- Open Pod Browser, verify that the header still looks and work properly on desktop view
- Switch to mobile view, check that only the hamburger icon is visible in the header. 
- Click on the hamburger and verify that the links work properly

### Screenshots:
- Desktop:
![image](https://user-images.githubusercontent.com/28412960/97328272-26237e80-1876-11eb-8e63-b6643a32c854.png)

- Mobile:
![image](https://user-images.githubusercontent.com/28412960/97328328-363b5e00-1876-11eb-8ea6-475556d1822a.png)
![image](https://user-images.githubusercontent.com/28412960/97328361-3d626c00-1876-11eb-852f-e468c6e1e7c1.png)


# Checklist
